ister.py

```diff
@@ -664,6 +674,30 @@ def set_hostname(template, target_dir):
         file.write(hostname)
 
 
+def set_mirror_url(target_mirror_url, target_dir):
+    """Writes custom mirror url to <target disk>/etc/swupd/mirror_contenturl
+    """
+    LOG.info("Setting custom mirror url")
+    path = '{0}/etc/swupd/'.format(target_dir)
+    if not os.path.exists(path):
+        os.makedirs(path)
+
+    with open(path + "mirror_contenturl", "w") as file:
+        file.write(target_mirror_url)
+
+
+def set_version_url(target_version_url, target_dir):
```

Comment:
Please call out that this is a mirror version url rather than the actual version url (since that exists as a file too).